qa/rpc-tests/test_framework/comptool.py

```diff
@@ -41,17 +41,32 @@ def wait_until(predicate, attempts=float('inf'), timeout=float('inf')):
 
     return False
 
+class RejectResult(object):
+    '''
+    Outcome that expects rejection of a transaction or block.
+    '''
```

Comment:
What could possibly go wrong here?
I can't see any risks merging as is.

FULL ACK from my side :+1: 

Bitcoin really needs more transaction throughput to grow!
We need to get a release ready in time!